June 12 release notes

gboone opened the pull request:
A few questions for the release notes this first time 'round.

1. Do we typically link off to a doc for the CVEs we remediated? I noticed in past releases, vulnerabilities fixed by upgrades were listed with the specific upgrade. (Also, were these in the presentation and I just missed them?)
2. Did I link to the right release notes for the different packages we upgraded? I found multiple sources and went with the CloudFoundry GitHub org's notes. I wasn't sure what to use for `stemcell`.
3. Anything I missed or misstated?

Commit:
Release notes from June 12, 2017

Commit:
Remove an unnecessary note

brittag commented on content/updates/2017-06-12-release-notes.md:
```diff
@@ -0,0 +1,25 @@
+
+### Fixed
+
+- We finished migrating all tenants from East/West into GovCloud! Thank you to all our customers for their patience and cooperation in making this move successful.
```
Our [most recent set of platform release notes](https://cloud.gov/updates/2017-06-01-release-notes/) said "cloud.gov documentation no longer provides instructions for using the deprecated East/West environment, because all customer applications have migrated to the GovCloud environment." so this seems redundant - do we need to mention it again?

gboone replied:
Hmm, good question. I feel like I remember @mogul celebrating everybody being out of E/W this time around. Is it that we _verified_ all customers are out of E/W and can begin shutting it down?

brittag commented on content/updates/2017-06-12-release-notes.md:
```diff
@@ -0,0 +1,25 @@
+### Fixed
+
+- We finished migrating all tenants from East/West into GovCloud! Thank you to all our customers for their patience and cooperation in making this move successful.
+- Updated tests, documentation and manual checks we perform to ensure we were ready to begin shutdown of the E/W environment.
```
This is internal rather than customer-facing, so customers don't need to hear about it.

gboone replied:
It's gone!

brittag commented on content/updates/2017-06-12-release-notes.md:
```diff
@@ -0,0 +1,25 @@
+
+- We finished migrating all tenants from East/West into GovCloud! Thank you to all our customers for their patience and cooperation in making this move successful.
+- Updated tests, documentation and manual checks we perform to ensure we were ready to begin shutdown of the E/W environment.
+- Clarified documentation about deployer accounts.
```
The helpful thing for customers to know is that actions by [space-deployer service accounts](https://cloud.gov/docs/services/cloud-gov-service-account/) previously said they were from `devops@gsa.gov`, which was confusing (and undocumented), and now they say they're from `service-account@cloud.gov` (which makes more sense and is documented). So it's good to be a bit more specific and also to link to the updated docs page.

brittag commented on content/updates/2017-06-12-release-notes.md:
```diff
@@ -0,0 +1,25 @@
+
+### Platform releases
+
+- [Fixed IP addresses for communicating with external apps](https://cloud.gov/docs/apps/static-egress/#cloud-gov-egress-addresses): Our customers often want cloud.gov apps that can talk to the outside and we now have an enabler to make that happen in a more secure way. Outbound traffic from cloud.gov comes from specific IP addresses to help customers open up a connection between cloud.gov and som ...
```
Ooh, there's something interesting here. This should be under its own heading - something like `### Changed` rather than `### Platform releases`. We use the "Platform releases" subheading for the general Cloud Foundry updates, rather than for the cloud.gov-specific updates. But this is easy to confuse, since we call cloud.gov a platform - so we probably need a new heading for the CF updates section.

The "Platform releases" section should be pretty similar to the last release notes, including the recommendation to restage apps.

Commit:
Adds a "changed" section and clarifies some things

brittag commented on content/updates/2017-06-12-release-notes.md:
```diff
@@ -0,0 +1,24 @@
+
+### Changed
+
+- [Fixed IP addresses for communicating with external apps](https://cloud.gov/docs/apps/static-egress/#cloud-gov-egress-addresses): Our customers often want cloud.gov apps that can talk to the outside and we now have an enabler to make that happen in a more secure way. Outbound traffic from cloud.gov comes from specific IP addresses to help customers open up a connection between cloud.gov and som ...
```
This description may be confusing in a couple of small ways - in this case, "fixed" means "static", but in a release notes post, "fixed" may read as "updated"! Also, cloud.gov apps can already talk to the outside, and this doesn't necessarily add substantial security (depending on context), but this change does enable better compatibility with common agency practices/preferences/policies.

brittag commented on content/updates/2017-06-12-release-notes.md:
```diff
@@ -0,0 +1,24 @@
+
+- Clarified documentation about deployer accounts: Previously actions from space-deployer service accounts displayed in logs as `devops@gsa.gov`. Now they display as the more specific and less confusing `service-account@gsa.gov`. [We've updated the documentation to reflect this](https://cloud.gov/docs/services/cloud-gov-service-account/).
+
+### Platform releases
```
We should bring in the more detailed platform release advice in the style of the previous ones: https://cloud.gov/updates/2017-06-01-release-notes/

gboone replied:
OK, cleared that up.

Commit:
Updated information about static IPs and the CF platform

brittag commented on content/updates/2017-06-12-release-notes.md:
```diff
@@ -0,0 +1,29 @@
+
+### Changed
+
+- [Static IP addresses for communicating with external apps](https://cloud.gov/docs/apps/static-egress/#cloud-gov-egress-addresses): Outbound traffic from cloud.gov now comes from specific IP addresses to help customers open up a connection between cloud.gov and outside data centers.
```
This anchor doesn't link to anything on https://cloud.gov/docs/apps/static-egress/#cloud-gov-egress-addresses - seems ok to just remove the anchor.

Also good to use relative links for links within the site - https://github.com/18f/cg-site#links - not required, but nice for future-proofing.

brittag commented on content/updates/2017-06-12-release-notes.md:
```diff
@@ -0,0 +1,29 @@
+
+### Fixed
+
+- Clarified documentation about deployer accounts: Previously actions from space-deployer service accounts displayed in logs as `devops@gsa.gov`. Now they display as the more specific and less confusing `service-account@gsa.gov`. [We've updated the documentation to reflect this](https://cloud.gov/docs/services/cloud-gov-service-account/).
```
We can remove "Clarified documentation about deployer accounts:" here since it was both a technical change (updating the email address) and a documentation change.

brittag commented on content/updates/2017-06-12-release-notes.md:
```diff
@@ -0,0 +1,29 @@
+- [RootFS cflinuxfs2 1.129.0](https://github.com/cloudfoundry/cflinuxfs2-rootfs-release/releases/tag/v1.129.0)
+- [Diego 1.18.0](https://github.com/cloudfoundry/diego-release/releases/tag/v1.18.0)
+- Stemcell 3312.28
+- 31 CVEs remediated
```
I think as a user I would read "31 CVEs remediated" and be a bit confused on whether this is important to me; in the previous release notes, we linked to the relevant USNs (which include the CVE lists) so that readers could review them - seems good to do that again if we can? I believe the way to gather this is to comb through the relevant release notes to find the relevant USN links.

Commit:
Edits to platform notes

Commit:
Remove "fixed"

Since the only item in this section was a documentation and technical
change, I've removed that change.

Commit:
Adds links to CVEs and USNs remediated

Commit:
Update platform notes and date
